package.json

```diff
@@ -35,7 +35,8 @@
     "tsdx": "^0.14.1",
     "tslib": "^2.3.1",
     "typescript": "^4.5.5",
-    "webpack": "^4.46.0"
+    "webpack": "^4.46.0",
+    "webpack-cli": "^4.10.0"
```

Comment:
Once v5 comes down the pipe, only `playground` will be using webpack, so I'm not sure this change is truly necessary